stages/org.osbuild.ovf

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/python3
+"""
+Create OVF descriptor and manifest
+
+Generates a OVF descriptor (xml) for a vmdk intended for vSphere.
+
+The OVF descriptor has minimal virtual hardware and no network.
+Hardware and network can be configured during or after importing
+into vSphere.
+
+Buildhost commands used: `qemu-img`.
+"""
+
+import json
+import os
+import subprocess
+import sys
+
+import osbuild.api
+from osbuild.util import checksum
+
+SCHEMA_2 = """
+"options": {
+  "additionalProperties": false,
+  "required": ["vmdk"],
+  "properties": {
+    "vmdk": {
+      "description": "Basename of the in-tree vmdk image, the vmdk should be in the root of the tree",
```

Comment:
Just a nitpick: the vmdk can be in fact (what the code allows):

- a basename of the image in the FS tree root
- a relative path (to the tree root) to the image
- an absolute path (basically wiping out the FS tree root path when joined) - this is IMO undesired.

Reply:
Changed, but kept the part after the comma. If the vmdk isn't in the root of the tree the tar stage might not produce a working ova.

Reply:
I think that it would be nicer if the stage failed instead of produced an incorrect OVA. The reason is that this could be a nightmare to debug later in composer if we use the stage in the wrong way. I would instead suggest to enforce the desired case either by `pattern` of the property or by checking the user-provided value in the code (e.g. if it is relative, absolute or just a basename) and failing in cases that we do not want to support.